### ISSUE TITLE
[question] Is the commandline use faster?

### ISSUE THREAD

**Issue by tmsd2001** [alicevision/meshroom#677]
On Linux Ubuntu, is the use Mashroom from commandline faster then the GUI Version?
And can I use a saved Meshroom mg file and run it from commandline?

**Comment by natowi** [comment]
> On Linux Ubuntu, is the use Mashroom from commandline faster then the GUI Version?

No. Meshroom is only a GUI for alicevision and does not affect the performance.

> And can I use a saved Meshroom mg file and run it from commandline?

Yes, use meshroom_compute.exe
